service/gcs/core/gcs/storage.go

```diff
@@ -185,14 +190,14 @@ func scsiControllerLunToName(osl oslayer.OS, controller, lun uint8) (string, err
 
 // scsiLunToName finds the SCSI device with the given LUN. This assumes
 // only one SCSI controller.
-func scsiLunToName(osl oslayer.OS, lun uint8) (string, error) {
-	return scsiControllerLunToName(osl, 0, lun)
+func (c *gcsCore) scsiLunToName(lun uint8) (string, error) {
+	return scsiControllerLunToName(c.OS, 0, lun)
 }
 
 // deviceIDToName converts a device ID (scsi:<lun> or pmem:<device#> to a
 // device name (/dev/sd? or /dev/pmem?).
 // For temporary compatibility, this also accepts just <lun> for SCSI devices.
-func deviceIDToName(osl oslayer.OS, id string) (device string, pmem bool, err error) {
+func (c *gcsCore) deviceIDToName(id string) (device string, pmem bool, err error) {
```

Comment:
This was V1 only. Marking that explicitly here. If we want this a separate change I can do that.